programs/local/customvm/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/ava-labs/avalanche-network-runner/local"
+	"github.com/ava-labs/avalanche-network-runner/network"
+	"github.com/ava-labs/avalanche-network-runner/utils"
+	"github.com/ava-labs/avalanche-network-runner/vms"
+	"github.com/ava-labs/avalanchego/utils/logging"
+	"github.com/spf13/pflag"
+	"github.com/spf13/viper"
+)
+
+const (
+	healthyTimeout = 2 * time.Minute
+	binaryPathKey  = "binary-path"
+	vmPathKey      = "vm-path"
+	genesisPathKey = "genesis-path"
+	subnetIDKey    = "subnet-id"
+	vmIDKey        = "vm-id"
+)
+
+var (
+	goPath            = os.ExpandEnv("$GOPATH")
+	defaultBinaryPath = fmt.Sprintf("%s%s", goPath, "/src/github.com/ava-labs/avalanchego/build/avalanchego")
+)
+
+type customVMConfig struct {
```

Comment:
This is actually multiple custom VM configs, not one, isn't it?
Would it make more sense to have this be
```
type customVMConfig struct {
	VMPath     string
	VMGenesis  string
	VMSubnet   string
	VMID       string
}
```
or similar? And then pass around `[]customVMConfig` where we now pass around just `customVMConfig`

Reply:
Of course it's possible, and also makes sense. But I don't think we gain much here. This is just the main for an example/program. We later in `run` create actual individual instances of `vms.CustomVM`, and pass this `customVMConfig` "around" just once to `run`.

I'd say we could keep it as-is.

Reply:
Because then the flag definition kinda gets confusing.

Now, we have e.g. `--vm-ids="1,2" --vm-paths="/path/1","/path/2"` etc.
The way you suggest, we would have to do something like `--vms="1 /path/1, 2 /path/2" ` and then it's not clear to the user for each vm which of the cli param is which.

For the time being the default use case will be one vm only anyway, which results in the neat `--vm-ids="1", --vm-paths="/path/1"`